portal.go

```diff
@@ -1252,6 +1252,11 @@ func (portal *Portal) HasRelaybot() bool {
 		return false
 	} else if portal.hasRelaybot == nil {
 		val := portal.bridge.Relaybot.IsInPortal(portal.Key)
+		if val == false {
+			user := portal.bridge.GetUserByJID(portal.bridge.Relaybot.User.JID)
+			inCommunity := user.addPortalToCommunity(portal)
+			val = user.CreateUserPortal(database.PortalKeyWithMeta{PortalKey: portal.Key, InCommunity: inCommunity})
```

Comment:
`addPortalToCommunity` is a no-op for the relaybot so calling it isn't necessary. There shouldn't be any need to call `GetUserByJID` either

```suggestion
			val = portal.bridge.Relaybot.CreateUserPortal(database.PortalKeyWithMeta{PortalKey: portal.Key, InCommunity: false})
```